src/controller/python/chip/ChipUtility.py

```diff
@@ -38,22 +38,20 @@ def VoidPtrToByteArray(ptr, len):
             v = bytearray(len)
             memmove((c_byte * len).from_buffer(v), ptr, len)
             return v
-        else:
-            return None
+        return None
 
     @staticmethod
     def ByteArrayToVoidPtr(array):
         if array != None:
             if not (isinstance(array, bytes) or isinstance(array, bytearray)):
                 raise TypeError("Array must be an str or a bytearray")
             return cast((c_byte * len(array)).from_buffer_copy(array), c_void_p)
-        else:
-            return c_void_p(0)
+        return c_void_p(0)
 
     @staticmethod
     def IsByteArrayAllZeros(array):
-        for i in range(len(array)):
-            if array[i] != 0:
+        for i in array:
```

Comment:
maybe this works:

```python
  return all(array, lambda x: x == 0)
```

Reply:
Resolving: update has a clear update purpose (convert range to array iterator), no need to nitpick on other potential changes.